DallasTemperature.cpp

```diff
@@ -454,7 +454,7 @@ void DallasTemperature::blockTillConversionComplete(uint8_t bitResolution) {
 }
 
 // returns number of milliseconds to wait till conversion is complete (based on IC datasheet)
-int16_t DallasTemperature::millisToWaitForConversion(uint8_t bitResolution) {
+uint16_t DallasTemperature::millisToWaitForConversion(uint8_t bitResolution) {

```

Comment:
Good one, this fixes mixed signed / unsigned math.
Are there argument to use **uint32_t**  instead of  **uint16_t**?
- millis() and micros() return uint32_t (most timing code I know use 32 bit)

Reply:
indeed, millis() returns a 32 bit unsigned (on Arduino at least) so there might be an argument to use that consistently, in this case the function return space would fit in 8 bit, so one might even consider that :) I went with keeping the spirit of the lib and did not opt for changing the layout :)

Reply:
I am 100% OK with 16 bit unsigned as 32 bit is indeed not needed, given the range of possible values. 
(solved)